server/athenian/api/precompute/accounts.py

```diff
@@ -285,14 +285,39 @@ async def create_teams(account: int,
     :return: Number of copied teams and the number of noticed bots.
     """
     _, num_teams = await copy_teams_as_needed(account, meta_ids, sdb, mdb, cache)
+    num_bots = await _ensure_bot_team(account, meta_ids, bots, prefixer, sdb, mdb)
+    return num_teams, num_bots
+
+
+async def _ensure_bot_team(
+    account: int,
+    meta_ids: Sequence[int],
+    bots: Set[str],
+    prefixer: Prefixer,
+    sdb: Database,
+    mdb: Database,
+) -> int:
     bot_team = await sdb.fetch_one(select([Team.id, Team.members])
                                    .where(and_(Team.name == Team.BOTS,
                                                Team.owner_id == account)))
     if bot_team is not None:
-        return num_teams, len(bot_team[Team.members.name])
+        return len(bot_team[Team.members.name])
+
     bots -= await fetch_bots.extra(mdb)
-    bots = set(chain.from_iterable(prefixer.user_login_to_node.get(u) for u in bots)) - {None}
+    bot_ids = set(chain.from_iterable(prefixer.user_login_to_node.get(u, ()) for u in bots))
```

Comment:
should we instead do `prefixer.user_login_to_node[u]` ? `u` comes from `node_bot` table and prefixer uses `api_users` so a KeyError is unexpected

Reply:
I had `[]` in the past and it raised a KeyError, I think. There are crazy mdb inconsistencies sometimes that fire in unexpected places.

Here the bots teams gets loaded from sdb and the user can add any crazy people inside.